### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by peterboro**
In this pull request I implemented the following changes:

- add the [JavaScript + HTML + CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js).
- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: html-css-js/.github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
+    name: Webhint
```
Comment (BertrandConxy) [inline html-css-js/.github/workflows/linters.yml:22]: - [ ] Your usual linters are passing and that's great 👏. However, you forgot to update the `linters.yml` file in order to include actions Github needs to parse in order to run eslint linter. I can see that it is here in the directory called `html-css-js` but right here, it is not reachable by Github. What I would suggest you do is to replace the `.github` directory you have here in **the html-css-js folder**, with the `.github` folder which is at **the root level of your website**. By doing this, you will have an updated `linters.yml` file which includes eslint rules to check for JavaScript syntax 😺.  After this, you can just remove that `html-css-js` folder as it is not needed anymore 💡

File: index.html [hunk 2]
```diff
@@ -46,27 +45,27 @@ <h3 class="connect">LET’S CONNECT</h3>
           <ul class="linkSocials">
             <li>
               <a href="twitter.com">
-                  <img src="./Images/SocialMediaIcons/Twitter.png" alt="Twitter" />
+                  <img src="/Images/SocialMediaIcons/Twitter.png" alt="Twitter" />
```
Comment (BertrandConxy) [inline index.html:48]: - [ ] [OPTIONAL] Your images look broken here in my browser. I don't know if they are visible on our side, you may check again locally but also on your deployed version. While I was trying to debug this, I realized that adding the dot before the `/` **will solve the bug** 😃. What I mean is that your src attribute looks like this, `src="./Images/SocialMediaIcons/Twitter.png" 💡 Please try to add it again if you find out that your images are broken. 🏃‍♂️

File: html-css-js/.github/workflows/linters.yml (hunk 1 above)
Comment (peterboro) [reply html-css-js/.github/workflows/linters.yml]: Thank you for the review, I've made the changes requested.

**Review by Chimwemwe-127001 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Hi @peterboro

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**